cmd/bench.go

```diff
@@ -301,7 +301,7 @@ func bench(ctx *cli.Context) error {
 	setup(ctx, 1)
 	/* --- Pre-check --- */
 	if ctx.Uint("block-size") == 0 || ctx.Uint("threads") == 0 {
-		return os.ErrInvalid
```

Comment:
This easier way is to change main():
```
func main() {
	err := Main(os.Args)
	if err != nil {
		log.Fatal(err)
	}
}
```

Reply:
Logger prints the line number and is usually helpful to trace the error. Besides, it's good to use only one way (fatal or return) to handle errors in commands.

Reply:
Main() is also in cmd/ package, we can use logger there, right?

Reply:
I meant changing `log` to `logger` in main should be enough.

Reply:
OK